tests/integration/targets/ecs_cluster/tasks/01_create_requirements.yml

```diff
@@ -0,0 +1,156 @@
+- name: ensure IAM instance role exists
+  iam_role:
+    name: ecsInstanceRole
```

Comment:
Is this role being used? I can't find anywhere that it is. If it is being used and I'm just missing it, it needs to be named `ansible-test-ecsInstanceRole` to work with the CI permissions. If it's not being used, we should just remove it from setup.